Functions.Templates/Templates/DurableFunctionsActivity-CSharp-1.x/metadata.json

```diff
@@ -12,7 +12,7 @@
   "userPrompt": [],
   "extensions": [
     {
-      "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask", "version": "1.8.7"
+      "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask", "version": "2.5.1"
```

Comment:
This is incorrect. This template specifically target v1 of durable. Updating the extension version will break the template. Also i don't think this template is currently being used.

Reply:
@soninaren  Sure. Do you suggest to not update the version for Durable and keep it 1.8.7?

Reply:
Sorry, got my answer from your another comment.